data.sql

```diff
@@ -14,4 +14,17 @@ VALUES
 	('Boarmon', '2005-06-7', 7, true, 20.4),
 	('Blossom', '1998-10-13', 3, true, 17),
 	('Ditto', '2022-05-14', 4, true, 22);
-	
\ No newline at end of file
+
+INSERT INTO owners(full_name, age)
+VALUES
+	('Sam Smith', 34),
+	('Jennifer Orwell', 19),
+	('Bob', 45),
+	('Melody Pond', 77),
+	('Dean Winchester', 14),
+	('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+	('Pokemon'),
+	('Digimon');
```

Comment:
- [ ] Laqstly here in `data.sql` we need to document the code that executes that all animals got assigned owners. Checkout the following example for the first one.

e.g
```
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
```